masonry/src/widget/mod.rs

```diff
@@ -61,7 +61,7 @@ use crate::{Affine, Size};
 // These are based on https://api.flutter.dev/flutter/painting/BoxFit-class.html
 /// Strategies for inscribing a rectangle inside another rectangle.
```

Comment:
Can we link to https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit in these docs?